prombench/docs/eks.md

```diff
@@ -0,0 +1,111 @@
+# Prombench in EKS
+
+Run prombench tests in [Elastic Kubernetes Service](https://aws.amazon.com/eks/).
+
+## Setup prombench
+
+1. [Create the main node](#create-the-main-node)
+2. [Deploy monitoring components](#deploy-monitoring-components)
+
+### Create the Main Node
+
+---
+
+- Create [security credentials](https://docs.aws.amazon.com/general/latest/gr/aws-sec-cred-types.html) on AWS. Put the credentials into a `yaml` file as follows:
+```yaml
+accesskeyid: <Amazon access key>
+secretaccesskey: <Amazon access secret>
+```
+- Create a [VPC](https://docs.aws.amazon.com/eks/latest/userguide/create-public-private-vpc.html) with public subnets.
+- Create a [Amazon EKS cluster role](https://docs.aws.amazon.com/eks/latest/userguide/service_IAM_role.html) with following policies:
+    - AmazonEKSclusterPolicy 
+- Create a [Amazon EKS worker node role](https://docs.aws.amazon.com/eks/latest/userguide/worker_node_IAM_role.html) with following policies:
+    - AmazonEKSWorkerNodePolicy
+    - AmazonEKS_CNI_Policy
+    - AmazonEC2ContainerRegistryReadOnly
+- Set the following environment variables and deploy the cluster.
+
+```shell
+export AUTH_FILE=<path to yaml credentials file that was created in the last step>
+export CLUSTER_NAME=prombench
+export ZONE=us-east-1
+export EKS_WORKER_ROLE_ARN=<Amazon EKS worker node IAM role ARN>
+export EKS_CLUSTER_ROLE_ARN=<Amazon EKS cluster role ARN>
+# By default SEPARATOR DeploymentVar is set to `,` but you can override it by exporting and
+# then passing it with the -v flag. It is used to split DeploymentVar into a slice.
+export SEPARATOR=, 
+export EKS_SUBNET_IDS=SUBNETID1,SUBNETID2,SUBNETID3
+
+../infra/infra eks cluster create -a $AUTH_FILE -v ZONE:$ZONE \
+    -v EKS_WORKER_ROLE_ARN:$EKS_WORKER_ROLE_ARN -v EKS_CLUSTER_ROLE_ARN:$EKS_CLUSTER_ROLE_ARN -v EKS_SUBNET_IDS:$EKS_SUBNET_IDS -v CLUSTER_NAME:$CLUSTER_NAME \
```

Comment:
we probably could make use of another `\` here as the line is getting a bit too long :)